docs/platforms/java/common/opentelemetry/setup/agentless.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: OpenTelemetry Agentless
+sdk: sentry.java.opentelemetry-agentless
+description: "Using OpenTelemetry with  sentry-opentelemetry-agentless."
+sidebar_order: 200
+---
+
+If you do not want to use our recommended <PlatformLink to="/opentelemetry/setup/agent">Java Agent</PlatformLink>, we also offer a dependency that allows you to use OpenTelemetry with Sentry.
+
+## Install
+
+<PlatformContent includePath="performance/opentelemetry-install/without-java-agent" />
```

Comment:
In this include, we show Information about how to install without agent, and have a paragraph that mentions that there is a separate dependency for spring boot. The link to which points to: `/platforms/java/guides/spring-boot/tracing/instrumentation/opentelemetry/`. But, in my opinion, should point to: `/platforms/java/guides/spring-boot/opentelemetry/setup/agentless/`. WDYT?